test/bats/public-ln-receive.bats

```diff
@@ -48,15 +141,28 @@ usd_amount=50
   payment_request="$(echo $invoice | jq -r '.paymentRequest')"
   [[ "${payment_request}" != "null" ]] || exit 1
 
+  # Setup subscription
+  variables=$(
+  jq -n \
+  --arg payment_request "$payment_request" \
+  '{"input": {"paymentRequest": $payment_request}}'
+  )
+  subscribe_to 'anon' 'ln-invoice-payment-status-sub' "$variables"
+  sleep 3
+  retry 10 1 grep "Data.*lnInvoicePaymentStatus.*PENDING" .e2e-subscriber.log
+
   # Receive payment
   lnd_outside_cli payinvoice -f \
     --pay_req "$payment_request" \
 
   # Check for settled
   retry 15 1 check_ln_payment_settled "$payment_request"
+
```

Comment:
this is using the query and not the subscription... is this on purpose? why not just wait for subscription?

Reply:
Changed it here to just wait for subscription. In the other tests here I use the query because it's quicker. Subscription only used once here to test the subscription explicitly https://github.com/GaloyMoney/galoy/pull/2971/commits/31dae18c8ce409f8088fc0e330e2c99085833937